### PR TITLE
Add T2T operation tests for all the compressed color formats

### PR DESCRIPTION
**Pull request by Jiawei-Shao**
-----

**[Reviews](https://github.com/gpuweb/cts/blob/main/docs/reviews.md) are outstanding for:**

- [x] WebGPU readability
- [x] TypeScript readability

### CODE REVIEW

**Comment by Jiawei-Shao** [comment]
@kainino0x @Kangz PTAL, thanks!

**Comment by kainino0x** [comment]
Oops, sorry I forgot to get to this today! Were out tomorrow so I'll get to this Monday.

**Comment by kainino0x** [comment]
> **[Reviews](https://github.com/gpuweb/cts/blob/main/docs/reviews.md) are outstanding for:**
> 
> * [x]  WebGPU readability
> * [ ]  TypeScript readability

Oops, sorry, I just realized I wrote this inverted. The intent is checked = good to go, unchecked = needs review of this aspect.

File: src/webgpu/api/operation/command_buffer/copyTextureToTexture.spec.ts [hunk 1]
```diff
@@ -331,63 +452,59 @@ g.test('color_textures,non_compressed,array')
...
-      .combine(poptions('dstCopyLevel', [0, 3]))
+      .combine(poptions('format', kCompressedTextureFormats))
+      .combine(poptions('copyBoxOffsets', F.kCopyBoxOffsetsFor2DArrayTextures))
+      .combine(poptions('srcCopyLevel', [0, 2]))
+      .combine(poptions('dstCopyLevel', [0, 2]))
   )
   .fn(async t => {
     const { textureSize, format, copyBoxOffsets, srcCopyLevel, dstCopyLevel } = t.params;
 
+    const extension: GPUExtensionName | undefined = kAllTextureFormatInfo[format].extension;
```
Comment (kainino0x) [inline src/webgpu/api/operation/command_buffer/copyTextureToTexture.spec.ts:504]: nit: If you use kCompressedTextureFormatInfo here, then you shouldn't need the assert (I think)

**Review by kainino0x — COMMENTED** [review]
Looks great! Just one nit

**Review by kainino0x — APPROVED** [review]

**Review by kainino0x — APPROVED** [review]

**Pull request merged**